CMakeLists.txt

```diff
@@ -957,7 +957,10 @@ IF(REDUCE_SWIG_DEBUG)
 ENDIF(REDUCE_SWIG_DEBUG)
 
 # For Swig and Unit-tests
-FIND_PACKAGE(Jinja2)
+IF(PythonModular OR LuaModular OR JavaModular OR RubyModular OR
+	OctaveModular OR CSharpModular OR RModular OR PerlModular OR ENABLE_TESTING)
+	FIND_PACKAGE(Jinja2)
```

Comment:
and please check whether the jinja2 was actually found... and then set a variable.